src/definitions/swap.ts

```diff
@@ -1,2 +1,3 @@
 export const CDP_LISTSWAPASSETS = 'cdp_listSwapAssets';
 export const CDP_GETSWAPQUOTE = 'cdp_getSwapQuote';
+export const CDP_GETSWAPTRADE = 'cdp_getSwapTrade';
```

Comment:
Thoughts on updating these to be snake case? 

```suggestion
export const CDP_LIST_SWAP_ASSETS = 'cdp_listSwapAssets';
export const CDP_GET_SWAP_QUOTE = 'cdp_getSwapQuote';
export const CDP_GET_SWAP_TRADE = 'cdp_getSwapTrade';
```